docs/end-to-end-test.md

```diff
@@ -0,0 +1,188 @@
+# End to end test using NOP algorithms
```

Comment:
it should be in the saps-quality-assurance repo, right?

Reply:
done